dashboard/config/course_offerings/music-jam.json

```diff
@@ -1,20 +1,20 @@
 {
   "key": "music-jam",
-  "display_name": "music-jam",
+  "display_name": "Music Lab: Jam Session",
   "is_featured": false,
   "assignable": true,
   "curriculum_type": "Hour of Code",
   "marketing_initiative": "HOC",
   "grade_levels": "2,3,4,5,6,7,8,9,10,11,12",
-  "header": null,
+  "header": "favorites",
   "image": "https://images.code.org/5aac6d5fbe2340096ef52dd6301bbbff-image (48).png",
   "cs_topic": "art_and_design,artificial_intelligence,programming",
   "school_subject": null,
   "device_compatibility": "{\"computer\":\"ideal\",\"chromebook\":\"ideal\",\"tablet\":\"ideal\",\"mobile\":\"not_recommended\",\"no_device\":\"incompatible\"}",
   "description": "Bring music and coding together with Music Lab: Jam Session! In this self-guided Hour of Code activity, students will remix tracks from artists like Sabrina Carpenter, Lady Gaga, and Shakira while learning coding basics and exploring AI-powered beats.",
   "professional_learning_program": null,
   "video": "https://www.youtube-nocookie.com/embed/6oK8392nq5c/?autoplay=1&enablejsapi=1&iv_load_policy=3&modestbranding=1&rel=0&showinfo=1&v=6oK8392nq5c&wmode=transparent",
-  "published_date": "2024-11-13 08:00:00 UTC",
+  "published_date": "2024-11-13 16:00:00 UTC",
```

Comment:
per slack convo, this approach is risky because it can cause merge conflicts in levelbuilder if any edits are made to the same unit there, because any update to that unit will cause a change in the published_date. it's also risky to undo the change to this line, because some things like PDF generation may depend on it.

No objections if you feel strongly about doing it this way, but my recommendation is to:
* make the change on levelbuilder
* go into levelbuilder and create a commit with only the changes you want to scoop
* push the changes to origin (it is ok if other commits are included)
* create a PR which cherry-picks that one commit into staging.

If you want to do it the way you're proposing, just make sure no one edits the same unit on levelbuilder until this PR is merged back into levelbuilder, or be prepared to deal with all the downstream effects of an overnight merge conflict.

Reply:
Thanks!  Updated PR is at https://github.com/code-dot-org/code-dot-org/pull/62413.